paddle/fluid/operators/reader/create_recordio_file_reader_op.cc

```diff
@@ -42,7 +41,7 @@ class RecordIOFileReader : public framework::FileReader {
     }
   }
 
-  void ShutdownImpl() override { scanner_.Reset(); }
+  void StartImpl() override { scanner_.Reset(); }
```

Comment:
`scanner_.Reset()` is not need to be invoked when `dctor`. So it is a `StartImpl`.